README.md

```diff
@@ -298,6 +306,24 @@ console.log("will raise an exception if not in script_hashes.yml!")
 <% end %>
 ```
 
+### Public Key Pins
+
+```
+config.hpkp = {
+  max_age: 60.days.to_i,   # max_age is a required parameter
+  include_subdomains: true, # whether or not to apply pins to subdomains
+  # Per the spec, SHA256 hashes are the only currently supported format.
+  pins: [
```

Comment:
Yeah, I definitely think it should be flexible enough to support alternate hash algos. I just thought the array of hashes syntax was a bit awkward. I'll leave up to @oreoshake since it is purely stylistic preference.